Plugin/ImaginationMedia/TmwGiftCard/ForceInvoicePlugin.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Bolt magento2 plugin
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @category   Bolt
+ * @package    Bolt_Boltpay
+ * @copyright  Copyright (c) 2017-2021 Bolt Financial, Inc (https://www.bolt.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+namespace Bolt\Boltpay\Plugin\ImaginationMedia\TmwGiftCard;
+
+use Bolt\Boltpay\Helper\Hook;
+
+/**
+ * Class ForceInvoicePlugin
+ * @package Bolt\Boltpay\Plugin\ImaginationMedia\TmwGiftCard
+ */
+class ForceInvoicePlugin
+{
+    /**
```

Comment:
Please add a brief description why this is needed.

Reply:
@danac-gs  I've added it. Can you check?